### ISSUE TITLE
Document stress tool

### ISSUE THREAD

**Issue by kfitzpatrick** [influxdata/docs.influxdata.com#138]
It would be great if the stress tool were documented in the web docs. Currently they only reside in the source code: https://github.com/influxdata/influxdb/tree/master/stress &  https://github.com/influxdata/influxdb/blob/master/stress/stress.toml

If I get time, I'll try to translate these into usable docs. Hopefully someone gets to it before me :P

**Comment by beckettsean** [comment]
@pauldix are we going to encourage people to benchmark 1.0 themselves? If so we need to make the stress tool easy to use.

**Comment by pauldix** [comment]
@beckettsean yeah, I think it should be well documented and easy to use. It's great for users to help them figure out sizing requirements.

**Comment by beckettsean** [comment]
@jwilder @mjdesa @jackzampolin @rossmcdonald @kostasb @mark-rushakoff 

Any thoughts on who should document the stress tool for end users? @rkuchan and I will happily turn your spaghetti notes into coherent prose, but we need a couple of pages of what it does and how in order for us to get started.

**Comment by jwilder** [comment]
I think @mjdesa @jackzampolin might be the most familiar with it.

**Comment by jackzampolin** [comment]
I'll sync up with @rkuchan to get it documented.

**Comment by beckettsean** [comment]
https://github.com/influxdata/influxdb/blob/master/stress/v2/README.md

Can we turn that into something in the docs?

@jackzampolin it's a great writeup for those familiar with the system, but I didn't quite understand how to launch it for evaluating my own cluster. Can you work up some step-by-step instructions to launch it against an InfluxEnterprise cluster, including what numbers to look at to determine if the cluster is well-provisioned?

**Comment by stevebang** [comment]
@timhallinflux It looks like `influx-stress` replaced the one described here and seems fairly well documented in the README (https://github.com/influxdata/influx-stress). Is this something we want to put into the 1.x docs now?
